packages/server/src/tests/utilities/api/table.ts

```diff
@@ -58,8 +58,13 @@ export class TableAPI extends TestAPI {
       .post(`/api/tables/${tableId}/migrate`)
       .send(data)
       .set(this.config.defaultHeaders())
-      .expect("Content-Type", /json/)
-      .expect(expectStatus)
+    if (res.status !== expectStatus) {
+      throw new Error(
+        `Expected status ${expectStatus} but got ${
+          res.status
+        } with body ${JSON.stringify(res.body)}`
+      )
+    }
```

Comment:
This made error messages from tests that call the migration endpoint significantly easier to debug. I would like functionality akin to this for all of our supertest expects, but I saw no easy way to do this generally.

Reply:
hmm - we might be able to overload the supertest object that the config returns, e.g. add it to what `this.request` does - I'll take a look at this!